pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/DDPMedicalRecordDataRequest.java

```diff
@@ -120,14 +71,23 @@ private static List<Integer> writeInstitutionInfo(Connection conn, InstitutionRe
 
             for (Institution institution : institutions) {
                 medicalRecordIds.add(MedicalRecordUtil.writeInstitution(conn, ddpParticipantId, instanceId,
-                        institution, instanceName));
+                        institution, ddpInstance.getName()));
             }
         } else {
             log.info("Institution list was empty for participant {}", ddpParticipantId);
         }
         return medicalRecordIds;
     }
 
+    public static void updateMedicalRecordLog(Connection conn, String ddpParticipantId, int instanceId) {
```

Comment:
This is needed temporarily as MedicalRecord services are moved from here to `MedicalRecordService`.